Jarvis.ino

```diff
@@ -42,31 +45,75 @@ void setup() {
 
   Log.begin();
   jarvis_sub->get();
-
 }
 
 void loop() {
   // Run the AdafruitIO Service
   io.run();
 
+  //if (Serial.available())
+  //  onSerialInput();
+
   // run the Jarvis desk interface
   Jarvis.run();
 
   Log.run();
 
   // Run the OTA Updater Service
   ota.loop();
-
 }
 
 // Handle messages from AdafruitIO
-void handlePreset(AdafruitIO_Data *data)
-{
+void handlePreset(AdafruitIO_Data* data) {
   Log.println(">MSG: ", data->feedName(), "=", data->toString());
 
   auto preset = data->toInt();
   flash(preset, 0.25);
 
   // Press the button
   Jarvis.goto_preset(preset);
+  Jarvis.report();
 }
+
+void onSerialInput() {
```

Comment:
This belongs in something called SerialServer maybe.  Then the main loop can call `SerialServer.setup()` and `SerialServer.run()` if desired.

Reply:
This was part of serial command interface while I was deving, but was removed as having the software serial and usb serial caused some bugs.